Documentation/gettingstarted/encryption.rst

```diff
@@ -224,26 +275,62 @@ Troubleshooting
 
    * Routing rules matching on fwmark:
 
-     .. code:: bash
+     .. code-block:: shell-session
 
-        ip rule list
+        $ ip rule list
         1:	from all fwmark 0xd00/0xf00 lookup 200
         1:	from all fwmark 0xe00/0xf00 lookup 200
         [...]
 
-   * Content of routing table 200
+   * Content of routing tables:
+
+    .. tabs::
+
+        .. group-tab:: GKE
+
+            Content of routing table 200:
+
+             .. code-block:: shell-session
+
+                $ ip route list table 200
+                local 10.60.0.0/24 dev cilium_vxlan proto 50 scope host
+                10.60.1.0/24 via 10.60.0.1 dev cilium_host
+
+        .. group-tab:: IPAM ENI mode
+
+            Content of routing table 200:
+
+            .. code-block:: shell-session
+
+               $ ip route list table 200
+               10.93.0.0/16 dev cilium_host mtu 9001
+               local 10.135.0.0/16 dev eth0 proto 50 scope host
 
-     .. code:: bash
+            Content of per-ENI routing table:
```

Comment:
Oh, and the indentation seems to be doing funny things here when rendered, could you double-check that?

Reply:
`make render-docs` will not pick up on odd indentation for what it's worth. In general sphinx seems to like 3-space indents (basically the next set of text is indented to the same indentation as the "code" part here.

I think the only way to really pick up on these is to render the docs and observe the generated output pages.